### PR TITLE
reference/sql: refine doc of expressions for pushdown

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
In this PR:
1. refine language
2. clarify the denotation of `表达式` under ##禁止特定表达式下推。

### What is the related PR or file link(s)? 
N/A

### Which version does your change affect? 
dev
I'll update v3.0 and v3.1 when these changes are approved.

### CODE REVIEW

**Comment by anotherrachel** [comment]
@fzhedu @yikeke PTAL, thanks!

File: dev/reference/sql/functions-and-operators/expressions-pushed-down.md [hunk 1]
```diff
@@ -21,25 +21,25 @@ category: reference
 
 ## 禁止特定表达式下推
 
-当函数的计算过程由于下推而出现不被期待的行为时，可通过将其拉入黑名单禁止下推来快速恢复业务。具体来讲，可以将上述支持下推的表达式拉入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止其下推。
+当函数的计算过程由于下推而出现异常时，可通过黑名单功能禁止其下推来快速恢复业务。具体而言，你可以将函数或运算符名加入黑名单 `mysql.expr_pushdown_blacklist` 中，以禁止特定表达式下推。
```
Comment (fzhedu) [inline dev/reference/sql/functions-and-operators/expressions-pushed-down.md]: 具体而言，你可以将**上述支持下推的**函数或者运算符名加入黑名单。。
（因为并不是所有的表达式都支持下推）

File: dev/reference/sql/functions-and-operators/expressions-pushed-down.md [hunk 2]
```diff
@@ -21,25 +21,25 @@ category: reference
...
 
 ### 加入黑名单
 
-执行以下步骤，可将一个或多个函数加入黑名单：
+执行以下步骤，可将多个函数或运算符同时加入黑名单：
```
Comment (yikeke) [inline dev/reference/sql/functions-and-operators/expressions-pushed-down.md]:
```suggestion
执行以下步骤，可将一个或多个函数或运算符加入黑名单：
```

File: dev/reference/sql/functions-and-operators/expressions-pushed-down.md [hunk 3]
```diff
@@ -94,11 +94,11 @@ tidb> explain select * from t where a < 2 and a > 2;
 
 > **注意：**
 >
-> - `admin reload expr_pushdown_blacklist` 只对执行该 SQL 的 TiDB server 生效，若需要集群中所有 TiDB server 生效，需要在每台 TiDB server 上执行该 SQL 语句。
+> - `admin reload expr_pushdown_blacklist` 只对执行该 SQL 语句的 TiDB server 生效，若需要集群中所有 TiDB server 生效，需要在每台 TiDB server 上执行该 SQL 语句。
 > - 表达式黑名单功能在 v3.0.0 及以上版本中支持。
-> - 在 v3.0.3 及以下版本中，不完全支持使用表达式原始名称文本（如 ">"，"+"，"is null"）添加黑名单，部分表达式在黑名单中使用别名。已支持下推的表达式中，表达式别名与原始名不同的，对应关系见下表（不区分大小写）。
+> - 在 v3.0.3 及以下版本中，黑名单功能不完全支持运算符原始名称文本（如 ">"，"+"，"is null"），部分运算符在黑名单中需使用别名。已支持下推的表达式中，别名与原始名不同的运算符见下表（不区分大小写）。
```
Comment (yikeke) [inline dev/reference/sql/functions-and-operators/expressions-pushed-down.md]:
```suggestion
> - 在 v3.0.3 及以下版本中，不支持将某些运算符的原始名称文本（如 ">"、"+" 和 "is null"）加入黑名单中，部分运算符在黑名单中需使用别名。已支持下推的表达式中，别名与原始名不同的运算符见下表（不区分大小写）。
```

File: dev/reference/sql/functions-and-operators/expressions-pushed-down.md [hunk 4]
```diff
@@ -94,11 +94,11 @@ tidb> explain select * from t where a < 2 and a > 2;
 
 > **注意：**
 >
-> - `admin reload expr_pushdown_blacklist` 只对执行该 SQL 的 TiDB server 生效，若需要集群中所有 TiDB server 生效，需要在每台 TiDB server 上执行该 SQL 语句。
+> - `admin reload expr_pushdown_blacklist` 只对执行该 SQL 语句的 TiDB server 生效，若需要集群中所有 TiDB server 生效，需要在每台 TiDB server 上执行该 SQL 语句。
```
Comment (yikeke) [inline dev/reference/sql/functions-and-operators/expressions-pushed-down.md]:
```suggestion
> - `admin reload expr_pushdown_blacklist` 只对执行该 SQL 语句的 TiDB server 生效。若需要集群中所有 TiDB server 生效，需要在每台 TiDB server 上执行该 SQL 语句。
```

File: dev/reference/sql/functions-and-operators/expressions-pushed-down.md (hunk 1 above)
Comment (anotherrachel) [reply dev/reference/sql/functions-and-operators/expressions-pushed-down.md]: Got it.

**Pull request merged**